src/components/questionary/questionaryComponents/TextInput/QuestionTemplateRelationTextInputForm.tsx

```diff
@@ -1,3 +1,4 @@
+import { Box } from '@material-ui/core';
```

Comment:
If we follow all other imports where we import defaults from specified component path this should be: `import Box from @material-ui/core/Box`. There is one in the `QuestionTextInputForm.tsx` file that should be changed as well I guess.